validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -140,6 +140,7 @@ def finalize_block(self, block_header):
 class BlockVerifier(BlockVerifierInterface):
     """DevMode BlockVerifier implementation
     """
+    # pylint: disable=useless-super-delegation
```

Comment:
All of the obvious ways of changing this result in further pylint errors (pylint seems to disagree with this design in general). Nontrivial changes are outside the scope of this PR. I've alerted @jjason.